engines/wintermute/detection_tables.h

```diff
@@ -68,8 +68,8 @@ static const PlainGameDescriptor wintermuteGames[] = {
 	{"pigeons",         "Pigeons in the Park"},
 	{"projectdoom",     "Project: Doom"},
 	{"projectjoe",      "Project Joe"},
-	{"reversion1",      "Reversion: The Escape"},
-	{"reversion2",      "Reversion: The Meeting"},
+	{"reversion1",      "Reversion: Chapter 1 - The Escape"},
```

Comment:
That's not how they appear in the Steam-store either.

Reversion - The Escape (1st Chapter)    (Where the buy button ignores the "1st Chapter" part)
Reversion - The Meeting (2nd Chapter)

What does the in-game window-title say?

I'm not against adding the numbers, but it needs to make sense and be consistent. (For instance the Carol Reed games are all numbered, even though the numbering is not ALWAYS applied in the games themselves).